### PR TITLE
update: add new bridge js, uuid for unique id in sample.

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
- Update bridge js asset due to this [update](https://github.com/rakutentech/ios-miniapp/pull/63) from js sdk.
- Update color accent from ios.
- Generating uuid in sample app.

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt [hunk 1]
```diff
@@ -62,7 +62,7 @@ class MiniAppDisplayActivity : BaseActivity() {
 
             launch {
                 viewModel.obtainMiniAppView(appId, object: MiniAppMessageBridge() {
-                    override fun getUniqueId() = "example_unique_id"
+                    override fun getUniqueId() = UUID.nameUUIDFromBytes(Build.DEVICE.toByteArray()).toString()
```
Comment (Climbatize) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt:65]: people with the same phone will have the UUID generated from the same array ?

Comment (minh-rakuten) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt:65]: That's right. There is no authentication for different users in our sample app at the moment so the sample app uses uuid generated from device.

Comment (corycaywood) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt:65]: Maybe we could use the Android ID instead? https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID

**Comment by minh-rakuten** [comment]
> LGTM, except my comment, but it's not that important as it's in the demo app

@Climbatize You're right so I've changed into ANDROID_ID.

**Pull request merged**